### DISCUSSION TITLE
Internal App Wallet using WalletConnect's SDKs

### DISCUSSION THREAD

**Discussion by LuisNeves05** [WalletConnect/walletconnect-monorepo#3027] [Developer Support]
Is there any way I can use WalletConntect's SDKs to create a local wallet in my app? Im building a POC for a social network and I want the UX to be really clean and not need to travel to other apps to sign transactions, so the idea is to create an internal wallet with transaction signing capabilities. Is that possible with the mentioned SDKs?

**Comment by crypblizz8** [comment]
Hey there @LuisNeves05, great to hear you are experimenting with WalletConnect.

We recommend using [Web3Wallet SDK](https://docs.walletconnect.com/2.0/web3wallet/about) to create your own Wallet so that you can handle all of the signing tx within your application.

Let us know if that is sufficient for you to get started!

**Reply by LuisNeves05** [reply]
Does it work with React Native and Expo GO?
